src/routes/_header-responsive.svelte

```diff
@@ -29,5 +30,10 @@
         Archival
       </NavigationLink>
     </IsCloudGuard>
+    <IsCloudGuard>
```

Comment:
Can this just go in the same guard as above?

Reply:
Moved to same cloud check